src/common/api/utils.ts

```diff
@@ -121,6 +129,8 @@ export const getChainTypeFromUrl = (networkUrl: string | undefined) => {
       return NetworkModes.Mainnet;
     case DEFAULT_TESTNET_SERVER:
       return NetworkModes.Testnet;
+    case DEFAULT_REGTEST_SERVER:
+      return NetworkModes.Regtest;
```

Comment:
This missing seemed to be the real issue ...the chain came in `undefined`.

Reply:
Shouldn't we return `NetworkMode.Testnet` here?